### DISCUSSION TITLE
Make it so you have to do a timing wizard if you start the game with bluetooth headphones

### DISCUSSION THREAD

**Discussion by AFasterSlowpoke** [ppy/osu#30796] [Ideas]
In Rotaeno, if you start the game with Bluetooth headphones instead of wired ones, it makes you do the timing wizard again, and it mutes all hitsounds. I think osu! should also implement such a system.

When you start osu! and it detects bluetooth headphones, i think it should make you go through a timing wizard (click every time you hear the snare drum or smth like that) and it will automatically set the offset from your results, along with muting all hitsounds (as if even the music offset was right and the player was hitting every note perfectly, hitsounds would still play too late and not in sync with the music). It should revert to the wired/speakers configuration once the Bluetooth is turned off, and vice versa.

While most rhythm gamers know to use wired headphones when possible, some laptops and phones do not have a headphone jack for whatever reason, and new players to osu! (and rhythm games) may not know beforehand that bluetooth really messes up the offset for rhythm games, much much more than other stuff (considering rhythm games are about perfect timing).

**Comment by peppy** [comment]
Sounds like something we'd definitely want to add to the new-user experience when we get to redoing that. It's a bit of a mess right now.
